src/main/java/uk/gov/hmcts/darts/common/entity/CourthouseEntity.java

```diff
@@ -49,7 +50,33 @@ public class CourthouseEntity extends CreatedModifiedBaseEntity {
         inverseJoinColumns = {@JoinColumn(name = "grp_id")})
     private Set<SecurityGroupEntity> securityGroups = new LinkedHashSet<>();
 
+    @ManyToMany
+    @JoinTable(name = "courthouse_region_ae",
+        joinColumns = {@JoinColumn(name = "cth_id")},
+        inverseJoinColumns = {@JoinColumn(name = "reg_id")})
+    private Set<RegionEntity> regions = new LinkedHashSet<>();
```

Comment:
having regions as a set might not be as efficient as making it a List, as then when you call getRegion(), you dont have to start a stream